Source/Client/Windows/ServerBrowser.cs

```diff
@@ -104,8 +104,15 @@ private void ReloadFiles()
             {
                 var saveFile = new SaveFile(Path.GetFileNameWithoutExtension(file.Name), false, file);
 
-                using (var stream = file.OpenRead())
-                    ReadSaveInfo(stream, saveFile);
+                try
```

Comment:
Wrapped call to read save info in a try/catch block to catch exceptions when reading. Looks good.